libnm/nm-device-ovs-bridge.c

```diff
@@ -26,12 +26,21 @@
 #include "nm-setting-ovs-bridge.h"
 #include "nm-setting-ovs-port.h"
 #include "nm-setting-connection.h"
+#include "nm-core-internal.h"
+
+enum {
+	PROP_0,
+	PROP_SLAVES,
+
+	LAST_PROP
+};
 
 /**
  * NMDeviceOvsBridge:
  */
 struct _NMDeviceOvsBridge {
 	NMDevice parent;
+	GPtrArray *slaves;
```

Comment:
slaves leaks. In both device types.

Reply:
Are you sure? They're cleared in dispose().